### DISCUSSION TITLE
Khul ke Bolo

### DISCUSSION THREAD

**Discussion by nalin360** [nalin360/khulKeBolo#12] [General]
It is a discussion tab for all the doubts and problem

**Comment by nalin360** [comment]
### **Logo Design Website :** 
- **[BrandCrowd](https://www.brandcrowd.com/maker/logos/page1?selectedAsset=true)**

![I](https://github.com/nalin360/khulKeBolo/assets/45825684/dd2b1d9e-4e9a-4a84-92d7-58424fc250b1)

- **[Tailor Brands](https://studio.tailorbrands.com/business/117237727/wizard/editor?backTo=logos&currentId=8639180773&logosStepId=702905645&origBrandVersionId=8639179469&originalIndex=0)**

![download](https://github.com/nalin360/khulKeBolo/assets/45825684/86894e97-e6c8-4012-94f6-8b3206c5886d)

**Reply by grecytembhare** [reply]
I'm also searching for app icon @nalin360

**Reply by nalin360** [reply]
Do yo like my suggestion

**Reply by grecytembhare** [reply]
It is nice but we can make it even better

**Reply by nalin360** [reply]
> I'm also searching for app icon @nalin360 

Stil you don't find any thing shame on you

**Reply by nalin360** [reply]
Still no reply @grecytembhare

**Reply by grecytembhare** [reply]
Will reply with some creative ideas

**Reply by nalin360** [reply]
I will deduct your salary , poor performance 
@grecytembhare

**Reply by grecytembhare** [reply]
Wait and watch

**Comment by nalin360** [comment]
@grecytembhare give response

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/afdbaa58-6281-42dd-b983-26dcfd1860b5)

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/3112604e-2d0b-48cb-9cc8-79bf337795a2)

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/5b787ff9-cfe0-43cf-94af-e810e75ce8c4)

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/b3f43360-c667-4661-be66-dfeebcd34505)

**Reply by nalin360** [reply]
not visible

**Comment by nalin360** [comment]
https://github.com/nalin360/khulKeBolo/assets/45825684/d21ac755-4741-4652-8072-e28b20812318

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/04a2beaa-684c-4a85-8c99-12032ecda23b)

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/aa5055ae-8f4b-493c-8ac7-33d7465787df)

**Comment by nalin360** [comment]
![Avery Davis](https://github.com/nalin360/khulKeBolo/assets/45825684/456b5b9f-984b-4d2e-87f1-96eb2012a087)

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/b121b9e9-1065-40fc-9414-fb8e01969b21)

**Comment by nalin360** [comment]
![Generic-Letter-B-Company-Logo-BrandCrowd-Logo-Maker-BrandCrowd](https://github.com/nalin360/khulKeBolo/assets/45825684/8c931ec4-ff80-46a5-b767-d0c8b156e237)

**Comment by grecytembhare** [comment]
![image](https://github.com/nalin360/khulKeBolo/assets/123292014/f1314f3c-2288-4288-9a8b-b67f9c0650de)

**Comment by nalin360** [comment]
![KKBl1](https://github.com/nalin360/khulKeBolo/assets/45825684/651cc526-d2e3-4d08-bc05-ec05edbf7460)

**Comment by nalin360** [comment]
### **SVG** 

![KKBl1svg](https://github.com/nalin360/khulKeBolo/assets/45825684/bcfce0e0-d5bc-478a-9306-a59284b13d1e)
